### DISCUSSION TITLE
Postgresql backup

### DISCUSSION THREAD

**Discussion by Stamka** [DefectDojo/django-DefectDojo#8334] [Q&A]
Hello everyone!

I have docker-compose postgres-redis DefectDojo instance.
How do the backup before update and after restore data?
I can't find full guide for this process.

**Comment by mtesauro** [comment]
If you have Postgres, then the easiest thing to do is just a pg_dump and use that command's output if you need to restore data. See https://www.postgresql.org/docs/current/backup-dump.html

There's loads of  Tutorial and documentation on pg_dump like https://www.digitalocean.com/community/tutorials/how-to-backup-postgresql-databases-on-an-ubuntu-vps.

The only other thing you may need to backup is any files uploaded to the configured "media" directory that DefectDojo users assuming that you've actually uploaded some files.
